cli/dinghy/dnsmasq.rb

```diff
@@ -7,10 +7,17 @@ class Dnsmasq
   RESOLVER_DIR = Pathname("/etc/resolver")
   RESOLVER_FILE = RESOLVER_DIR.join("docker")
 
-  attr_reader :machine
+  attr_reader :machine, :resolver_file, :dinghy_domain
 
   def initialize(machine)
     @machine = machine
+    @dinghy_domain = "docker"
```

Comment:
just call `self.dinghy_domain = "docker"` here to avoid repeating these lines of code